DEVELOPER.md

```diff
@@ -44,13 +45,67 @@ Please run the testing targets locally before you push a PR.
 
 Notes on how to test an existing CR Database.
 
-kubectl exec into a pod
+# Development
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [kustomize](https://kustomize.io/)
+
+The rest of dependencies are packed into Docker images which are executed from the [](Makefile)
+
+### Running the operator in GCP
 
 ```
-cockroach sql
-CREATE TABLE bank.accounts (id INT PRIMARY KEY, balance DECIMAL);
-INSERT INTO bank.accounts VALUES (1, 1000.50);
-SELECT * FROM bank.accounts;
+git clone https://github.com/cockroachdb/cockroach-operator.git
+export CLUSTER=test
+# create a gke cluster
+./hack/create-gke-cluster.sh -c $CLUSTER
+
+# build the image locally and push it to your image repo
+# record the output of the command, as it will tell you the image to use with kustomize
+./hack/push-operator-gcr.sh
+
+# apply the operator
+./hack/apply-operator.sh -c $CLUSTER
+
+# validate the the operator is running
+alias kubectl=k
+k get po
+
+# install a basic example
+./hack/apply-crdb-example.sh -c $CLUSTER
+```
+You now should have a basic crdb running. You can kubectl exec into a pod and test the database.
+
+Clean up the cluster
+
+```
+# delete the example
+./hack/delete-crdb-example.sh -c $CLUSTER
+
+# delete the operator
+./hack/delete-operator.sh -c $CLUSTER
+
+# If you're still using the gke cluster, you can delete persistent volumes and persistent volume claims. It is not recommended to do this in production. Use --help for details.
+kubectl delete pv,pvc --help
+
+# delete the cluster
+# note this is async, and the script will complete without waiting the entire time
+./hack/delete-gke-cluster.sh -c $CLUSTER
```

Comment:
This is done via `make k8s/delete` now.

Reply:
Is this the full command or do you still need to specify the cluster name?

Reply:
@chrislovecnm `make k8s/delete` did not work for me and my pods are still running. Should I file a bug? This was returned at the end:

```
error: no server found for cluster "gke_chris-love-operator-playground_us-central1-a_test"
```